manifests/repo/ubuntu.pp

```diff
@@ -27,7 +27,8 @@
     '5.5' => 'ondrej/php',
     '5.6' => 'ondrej/php',
     '7.0' => 'ondrej/php',
-    '7.1' => 'ondrej/php'
+    '7.1' => 'ondrej/php',
+    '7.2' => 'ondrej/php'
```

Comment:
Just a minor improvement, you can add a `,` to the last line too (it's not json luckily :P)